Test/task3/task3_solution.py

```diff
@@ -0,0 +1,13 @@
+def sum_numbers(n):
+    total = sum(range(1, n + 1))
+    expected = n * (n + 1) // 2
+
+    assert total == expected, f"Ошибка: ожидалось {expected}, а получено {total}"
+
+    return total
```

Comment:
you do not need to count and assert it here
```suggestion
def sum_numbers(n):
    expected = n * (n + 1) // 2

    return expected
```